src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionResolveEndpoint.cs

```diff
@@ -121,27 +127,45 @@ public Task<CompletionItem> Handle(CompletionItem completionItem, CancellationTo
                 case RazorCompletionItemKind.TagHelperAttribute:
                     {
                         var descriptionInfo = associatedRazorCompletion.GetAttributeCompletionDescription();
+                        if (descriptionInfo == null)
```

Comment:
I can't imagine this ever happening in practice. It's mostly there because of `#nullable enable`; however, if it were to happen it'd be a misconfiguring of a RazorCompletionItem OR the completion list cache got cleared somehow so we couldn't re-lookup the right completion item.